src/frontend/app/shared/components/stepper/steppers/steppers.component.html

```diff
@@ -2,23 +2,16 @@
   <div class="steppers" *ngIf="steps && steps.length">
     <div class="steppers__inner">
       <div class="steppers__headers" *ngIf="steps.length > 1">
-        <div *ngFor="let step of steps; let i = index" (click)="setActive(i)" [ngClass]="{'steppers__header--active': step.active || step.complete}" class="steppers__header">
+        <div *ngFor="let step of steps; let i = index" [ngClass]="{'steppers__header--active': step.active || step.complete}" class="steppers__header">
           <div class="steppers__header-inner" *ngIf="steps.length !== 1">
             <!-- Blocked -->
             <mat-spinner *ngIf="step.blocked; else notBlocked" diameter="20"></mat-spinner>
             <!-- Not Blocked -->
             <ng-template #notBlocked>
-              <!-- Skipped -->
-              <span *ngIf="step.skip; else notSkipped">
-                <app-dot-content class="buildpack-card__title-dot" [disabled]="true"><div>{{ i + 1 }}</div></app-dot-content>
-              </span>
-              <!-- Not Skipped -->
-              <ng-template #notSkipped>
-                <app-dot-content class="buildpack-card__title-dot">
-                  <mat-icon class="steppers__header-complete" *ngIf="step.complete">{{ this.getIconLigature() }}</mat-icon>
-                  <div *ngIf="!step.complete ">{{ i + 1 }}</div>
-                </app-dot-content>
-              </ng-template>
+              <app-dot-content class="buildpack-card__title-dot" [disabled]="step.skip">
+                <mat-icon class="steppers__header-complete" *ngIf="step.complete">{{ this.getIconLigature() }}</mat-icon>
```

Comment:
We should rename the CSS class here.. `buildpack-card`, there is no rule for it in the Steppers Component anyway